plugins/generic/tinymce

```diff
@@ -1 +0,0 @@
-Subproject commit 949badbccc42f10d732b71df7d0fb1e9b585b565
```

Comment:
ah no, I will push everything again but in the same branch i.e. in the same pull request...